### ISSUE TITLE
interop with alexpevzner/sane-airscan fails

### ISSUE THREAD

**Issue by markosjal** [SimulPiscator/AirSane#76]
Just FYI , testing interoperability with alexpevzner/sane-airscan Multi Page scanning from ADF fails. 

Please see attached wireshark pcap . Sorry because it is a 'localhost' connection it can be hard to follow (same IP to/from Same IP)

At packet number 1031 you see "409 Unknown Reason" . I believe that is the point where it all falls apart in Document scanner 3.36.3

This was tested on a new install of Linux Mint 20.2 Uma base: Ubuntu 20.04 focal. AirSane Was installed right after your addition of "Serial Number" in xml.

The capture file is here
http://portlandtel.com/pcap/eSCLFailpcapng.zip

Bug report for alexpevzner/sane-airscan is here 
https://github.com/alexpevzner/sane-airscan/issues/171

**Comment by SimulPiscator** [comment]
409 means "Hardware Conflict" and, in this case, indicates that SANE reports an empty ADF.

**Comment by markosjal** [comment]
But not empty. There were 4 pages loaded in adf . First scans successfully and  does second page which displays.second , then seems to  disappear from Scan app at about the time of this error. In other words This second page appears then after the display of the second page disappears although scanner is very fast and all scanning from ADF may have actually completed .

**Comment by SimulPiscator** [comment]
ADF support is broken in AirSane. @ShadesJeff has created a fork that is supposed to fix it: https://github.com/ShadesJeff/AirSane. Maybe you give that a try?

**Comment by markosjal** [comment]
Is ADF still broken?

**Comment by SimulPiscator** [comment]
ADF code has been reworked in commit [26d70d4](https://github.com/SimulPiscator/AirSane/commit/26d70d4714aa678bde694f4b1084f15ce053cda7)

**Comment by SimulPiscator** [comment]
Closed due to inactivity
